Fitness/Controllers/HomeController.swift

```diff
@@ -13,35 +13,38 @@ import SnapKit
 import UIKit
 
 enum SectionType: String {
+    case checkIns = "DAILY CHECK-INS"
     case allGyms = "ALL GYMS"
-    case lookingFor = "I'M LOOKING FOR..."
-    case pros = "LEARN FROM THE PROS"
     case todaysClasses = "TODAY'S CLASSES"
+    case pros = "LEARN FROM THE PROS"
 }
 
 enum SectionInsets {
```

Comment:
We could make this a function of `SectionType` and have it switch on the enum and return these insets. The benefits of this is the switch statement will force it to be exhaustive on the enum in case we ever add a new one, someone won't forget to add insets.

Reply:
Sounds good!